nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/FetchElasticsearchHttp.java

```diff
@@ -214,7 +214,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             logger.debug("Fetching {}/{}/{} from Elasticsearch", new Object[]{index, docType, docId});
 
             // read the url property from the context
-            final String urlstr = StringUtils.trimToEmpty(context.getProperty(ES_URL).evaluateAttributeExpressions().getValue());
+            final String urlstr = StringUtils.trimToEmpty(getESUrl());
```

Comment:
The `getESUrl()` method calls `trim()` on the value being returned, so do we really need `StringUtils.trimToEmpty` or `.trim()` calls in all the individual processors (I'd imagine not)?